src/main/app/fees/feesClient.ts

```diff
@@ -34,49 +35,48 @@ export class FeesClient {
    * @returns {Promise.<number>} promise containing the fee amount in pounds
    */
   static async calculateHearingFee (claimValue: number): Promise<number> {
-    const fee: CalculationOutcome = await this.calculateFee(hearingFeeCode, claimValue)
-    return MoneyConverter.convertPenniesToPounds(fee.amount)
+    return this.calculateFee(hearingFeeEvent, claimValue, defaultChannel)
+      .then((outcome: FeeOutcome) => (outcome.amount))
   }
-
   /**
-   * Calculates the fee based on fee code and amount
+   * Calculates the fee based on fee event and amount
    *
-   * @param feeCode which fee category to use
+   * @param eventType which fee event to use
    * @param amount amount in pounds
-   * @returns {Promise.<CalculationOutcome>} promise containing the calculation outcome (including fee amount in pennies)
+   * @returns {Promise.<FeeOutcome>} promise containing the Fee outcome (including fee amount in GBP)
    */
-  static async calculateFee (feeCode: string, amount: number): Promise<CalculationOutcome> {
-    ClaimValidator.claimAmount(amount)
-    const amountInPennies = MoneyConverter.convertPoundsToPennies(amount)
-    if (amountInPennies <= 0) {
-      throw new Error(`Amount must be at least 1 penny, amount was: ${amountInPennies}`)
+  static async calculateFee (eventType: string, amount: number, channel: string): Promise<FeeOutcome> {
+    if (StringUtils.isBlank(eventType)) {
+      throw new Error('Fee eventType is required')
     }
-
-    const fee: object = await request.get(`${feesUrl}/range-groups/${feeCode}/calculations?value=${amountInPennies}`)
-    return plainToClass(CalculationOutcome, fee)
+    ClaimValidator.claimAmount(amount)
+    const feeUri: string = `${feesUrl}/fees-register/fees/lookup?service=${service}&jurisdiction1=${jurisdiction1}&jurisdiction2=${jurisdiction2}&channel=${channel}&event=${eventType}&amount_or_volume=${amount}`
+    const fee: object = await request.get(feeUri)
+    return plainToClass(FeeOutcome, fee)
   }
 
   /**
    * Get the issue fee range group
-   * @returns {Promise.<RangeGroup>} promise containing the range group (including fee amounts in pennies)
+   * @returns {Promise.<FeeRange>} promise containing the range group (including fee amounts in GBP)
    */
-  static async getIssueFeeRangeGroup (): Promise<RangeGroup> {
-    return this.getRangeGroup(issueFeeCode)
+  static async getIssueFeeRangeGroup (): Promise<FeeRange[]> {
+    return this.getRangeGroup(issueFeeEvent, onlineChannel)
```

Comment:
Please mention in JSDoc that this function returns fee values used in online channel so no surprise when somebody used that function for getting issue fees issued on paper.